doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -321,6 +309,119 @@ Error list (HTTP response code in paranthesis):
 * UnsupportedMediaType (415). Request content type is not supported.
 * InvalidModification (422). Some piece of information is missing in payload.
 
+## Geospatial properties of entities
+
+The geospatial properties of a context entity can be represented by means of regular context attributes.
+The provision of geospatial properties enables the resolution of geographical queries.
+
+Two different syntaxes must be supported by compliant implementations: 
+
+* *Simple Location Format*. It is meant as a very lightweight format that developers and users can quickly
+and easily add to their existing entities with little effort.
+
+* *GeoJSON*.  [GeoJSON](https://tools.ietf.org/html/draft-butler-geojson-06)
+is a geospatial data interchange format based on the JavaScript Object Notation (JSON). GeoJSON provides greater flexibility
+allowing the representation of complex geospatial features.
+
+Additionally, geospatial properties might be declared by means of [civic addresses](https://schema.org/PostalAddress),
+but in that particular case context producers will have to implement their own geocoding procedures,
+so that geographical queries can be resolved.
+
+Client applications are responsible for defining which entity attributes convey geospatial properties (by providing
+an appropriate NGSI attribute type). Typically it will be an entity attribute named `location`,
+but nothing prevents use cases where an entity contains more than one geospatial attribute. For instance,
+locations specified at different granularity levels or provided by different location methods with different accuracy. 
+Nonetheless, it is noteworthy the fact that spatial properties
+need special indexes which can be under resource constraints imposed by backend databases.
+Thus, implementations may raise errors when spatial index limits are exceeded.
+The recommended HTTP error code for those situations is ``413`` (Request entity too large).
+
+### Simple Location Format
+
+Simple Location Format supports basic geometries ( *point* , *line* , *box* , *polygon* ) and covers the typical
+use cases when encoding geographical locations. It has been inspired by [GeoRSS Simple](http://www.georss.org/simple.html).
+
+It is noteworthy that the Simple Location Format is not intended to represent complex positions on Earth surface.
```

Comment:
I think there would be many use cases willing to use simple location format to represent locations in Earth surface and this sentence could be discouraging for them. If the only problem is the altitude, I'd suggest to state it clearly (instead of speaking about "complex position" in general without define precisely what we understand by "complex" or, in other words, what is "simple" and what is "complex" in this context).